src/ws/test-remotectlcertificate.c

```diff
@@ -188,6 +189,15 @@ test_valid_selfsigned (TestCase *test,
   dir = g_dir_open (test->cert_dir, 0, &error);
   g_assert_no_error (error);
   fname = g_dir_read_name (dir);
+  if (sscg_path)
```

Comment:
this is gross, but I guess it works...

could you be convinced to try a different approach here?

 - iterate the directory, appending to a gstring, then compare both orders?
 - add items to a set-mode hash table?
 - have two 'seen' bools for each of the filenames we are expecting, set them from a loop, check them after?
 - some sorted variant of any of the above?